build-zip.sh

```diff
@@ -10,21 +10,14 @@ CONFIG_FILE="${ZIP_FLAVOUR}_config.txt"
 
 ADDOND_FILE='70-microg.sh' #common to all flavours
 
-apps_config() {
-  sed -e '/^#/d' "$CONFIG_FILE"
-}
-
 #_______________________________________________________________________________
 #                             Exported functions
 function fetch() {
-  local URL=$1
-  local FILENAME=$2
+  local URL="$1"
+  local FILENAME="$2"
 
-  wget \
-  --no-verbose \
-  --output-document=$FILENAME \
-  $URL
-} && export -f fetch
+  curl -o "$FILENAME" "$URL"
```

Comment:
Why using cURL? I'm not against, but downloading a file is a simple usecase that wget supports just fine.

Also, observe how we used command flags/options. Longer names are preferred for readability.

Reply:
Because the busybox's `sh` seems to use the built-in `wget` that does not support https

Reply:
Use Termux's `wget`, as stated in https://github.com/termux/termux-packages/issues/2423

Reply:
More scripts use curl